### ISSUE TITLE
VRCFury locked to 1.1013.0

### ISSUE THREAD

**Issue by LauraRozier** [LastationVRChat/NSFW-GoLoco#13]
### The project version

v1.8.3-NSFW-1.1.0

### Which Unity Version are you using?

2022.3.22f1

### Describe your issue.

VRCFury is locked to 1.1013.0, currently we're at 1.1071.0, kindly stop capping the VRCFury release, there is no reason for the version lock.

### Steps to reproduce the problem:

```bash
1. Add it to VCC
2. Add the package
3. Get locked to VRCFury 1.1013.0
4. Cry because you're missing a boatload of fixes
```

### Have you modified the project?

No

### Did the issue happen in-game or on Unity Editor?

Unity Editor

**Comment by KuryKat** [comment]
**Thank you for bringing this to our attention.**

To clarify, the current version isn't locked—users can still manually update to the latest version of VRCFury if needed. We're actively working on expanding the supported version range in an upcoming release. However, due to the rapid updates to VRCFury, it's challenging to guarantee stability with every new version, so we locked it to the version we initially tested with to ensure consistency.

We appreciate your patience and understanding as we work toward a more flexible solution!

**Comment by KuryKat** [comment]
Hey there!

We’ve just pushed a fix to address the dependency version range issue that was causing compatibility problems with VRCFury in the VPM package. The version lock has been fixed to ensure smoother integration with newer releases of the dependency!

We know it’s been quite some time since the last update, and we sincerely apologize for the delay in addressing this relatively small issue. This side project had to take a backseat while we worked on higher-priority tasks, including a full backend rewrite for our main systems.

Thank you so much for your patience and understanding! We’re committed to keeping this package up-to-date and reliable for all users. The patch will be included in the upcoming v1.1.1 release. 💜
